src/sentry/static/sentry/app/views/issueList/createSavedSearchButton.tsx

```diff
@@ -80,12 +91,12 @@ class CreateSavedSearchButton extends React.Component {
     }
   };
 
-  handleChangeName = val => {
-    this.setState({name: val});
+  handleChangeName = (val: FieldOnChangeParameters) => {
+    this.setState({name: String(val)});
```

Comment:
I believe that in this case the val will always be of type string, so I would only cast because it's a type issue.